src/foam/nanos/controller/ApplicationController.js

```diff
@@ -549,6 +549,8 @@ foam.CLASS({
       } else if ( this.theme ) {
         this.window.location.hash = this.theme.defaultMenu;
       }
+
+      this.__subContext__.localSettingsDAO.put(foam.nanos.session.local_settings.LocalSetting.create({name: 'homeDenomination', value: 'USD'}));
```

Comment:
local_settings -> localSettings (for package name).
And in this case fine will just putting it in the Session package directly.